tests/core/test_structured_data_signing.py

```diff
@@ -233,9 +236,9 @@ def test_invalid_structured_data_invalid_abi_type():
         "tests/fixtures/invalid_message_invalid_abi_type.json"
     ).read()
     invalid_structured_data = json.loads(invalid_structured_data_string)
-    with pytest.raises(TypeError) as e:
+    with pytest.raises(ABITypeError) as e:
```

Comment:
Could use the `match` keyword instead of the `assert`: https://docs.pytest.org/en/latest/assert.html

(Obviously just an optional suggestion to fix broken windows, since this pattern was already here pre-PR)

Reply:
Giving it a shot

Reply:
Nope, had some weird issue.